apps/backend/src/mcp/prompts/container_workflows.py

```
@@ -0,0 +1,323 @@
+"""
+Container lifecycle management prompts for infrastructure workflows.
+
+These prompts create comprehensive workflows for managing Docker containers
+across the infrastructure, leveraging multiple MCP tools for complete lifecycle management.
+"""
+
+
+def deploy_application_stack(
+    hostname: str, 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)